omero/sysadmins/customization.rst

```diff
@@ -1,13 +1,18 @@
-Customize OMERO clients
+OMERO.web customization
 =======================
 
-The OMERO clients offer a flexible user interface that can be customized.
+The OMERO.web offer a flexible user interface that can be customized.
 The sections below describe how to set up these features.
 
 Note that depending on the deployment choice, OMERO.web will not activate
 configuration changes until gunicorn is restarted using ``bin/omero web
 restart``.
 
+Note that the command ``bin/omero`` used throughout this page refers to
```

Comment:
I'm concerned about needing to repeat this box multiples times. It's currently in 2 locations and could equally be added to at least `omeroweb-upgrade`.

Reply:
I agree in the current state of the pages, I thought adding in both places is the less disruptive unless we split everything with and without server

Reply:
Discussed with @joshmoore and @sbesson briefly 
an option will be to teach sphinx to make command clickable or display
I will remove it from customization page and only keep it in the install page